### ISSUE TITLE
Epithelial corneal edema

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10872]
Preferred term label:
Epithelial corneal edema

Synonyms
Corneal epithelial edema

Definition (free text, please give PubMed ID)
Swelling of the corneal epithelium caused by the accumulation of fluid within epithelial cells or intercellular spaces, often resulting in blurred vision and corneal haze.

Parent term (use hpo.jax.org/app)
Corneal edema (HP:0002181)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
None identified

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
this already existed: Corneal epithelial edema HP:6001182
I added the synonym, thanks!
